core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionNetworkOutput.java

```diff
@@ -81,6 +82,11 @@ public static TransmissionNetworkOutput create() {
     }
 
     public static TransmissionNetworkOutput create(String endpoint) {
+        if (!ClassDataUtils.INSTANCE.isClassExists("org.apache.http.conn.HttpClientConnectionManager")) {
+            String errorMessage = String.format("HttpClient Jars mismatch: please make sure version 4.3.1 of HttpClient is used");
```

Comment:
I think that we should be good with v4.3.
Can you please verify it? (you can simply change the version we're using in build.gradle and see that things don't get broken)